packages/circuits/circom.config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "circom": "../../../../.cargo/bin/circom",
```

Comment:
I wonder if we could easily populate this from `which circom`

`circom-helper` doesn't seem to have an option to set this directly, but we could replace this with a script that creates the config file on demand.